lms/urls.py

```diff
@@ -608,6 +609,9 @@
 
 if settings.DEBUG:
     urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+    urlpatterns += static(
```

Comment:
Maybe you can explain it to me in words sometime. I'm not clear how the value of settings.DEBUG is related to setting up these urlpatterns.